example/basic_example.py

```diff
@@ -96,73 +98,69 @@ def parse_arguments():
             "OneAgent endpoint.")
 
     # set up OpenTelemetry for export:
+    # This call sets up the MeterProvider, with a PeriodicExportingMetricReader that exports every 5000 ms
+    # and the Dynatrace exporter exporting to args.endpoint with args.token
     logger.debug("setting up global OpenTelemetry configuration.")
-    metrics.set_meter_provider(MeterProvider())
-    meter = metrics.get_meter(splitext(basename(__file__))[0])
+    _metrics.set_meter_provider(MeterProvider(
+        metric_readers=[PeriodicExportingMetricReader(
+            export_interval_millis=5000,
+            exporter=DynatraceMetricsExporter(args.endpoint, args.token,
+                                              prefix="otel.python",
+                                              export_dynatrace_metadata=args.metadata_enrichment))]))
+
+    meter = _metrics.get_meter(splitext(basename(__file__))[0])
 
-    logger.info("setting up Dynatrace metrics exporting interface.")
     exporter = DynatraceMetricsExporter(args.endpoint, args.token,
                                         prefix="otel.python",
                                         export_dynatrace_metadata=
                                         args.metadata_enrichment)
 
     logger.info("registering Dynatrace exporter with the global OpenTelemetry"
                 " instance...")
-    # This call registers the meter and exporter with the global
-    # MeterProvider set above. All instruments created by the meter that is
-    # registered here will export to the Dynatrace metrics exporter. It is a
-    # good idea to keep a reference to the meter (e. g. in a global variable)
-    # in order to create instruments anywhere in the code that all export to
-    # the same Dynatrace metrics exporter.
-    metrics.get_meter_provider().start_pipeline(meter, exporter, args.interval)
 
     logger.info("creating instruments to record metrics data")
     requests_counter = meter.create_counter(
         name="requests",
         description="number of requests",
-        unit="1",
-        value_type=int
+        unit="1"
     )
 
-    requests_size = meter.create_valuerecorder(
+    requests_size = meter.create_histogram(
         name="request_size_bytes",
         description="size of requests",
         unit="byte",
-        value_type=int,
```

Comment:
As far as I can tell, yes. 
Everything that I could find in the o[pentelemetry-python](https://github.com/open-telemetry/opentelemetry-python) repository referencing it looks like it is left over from previous versions.

Reply:
I wonder of we still need the float vs int values then anymore.

Reply:
As far as I can tell, yes. I cannot find any notable references to it in the `opentelemetry-python` repository anymore.

Reply:
Do you mean float vs int values in the example, or what we discussed [here](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/22#discussion_r826785982)? :thinking:

Reply:
I guess it goes in the same direction. If OTel Python decided to only ever use floats, what is the point of distinguishing here? I have not double checked if that is the case however!

Reply:
It will use whichever type Python decides to be the datatype based on the values used on the instrument, so if only `int`s are used for the instrument, it will be `int`, otherwise the exported value will be `float`. 

For `basic_example.py`, I think the important part is that we're showing the different instrument types being exported.